src/lib/openjp2/t1.c

```diff
@@ -1003,130 +1197,186 @@ static void opj_t1_enc_clnpass(
 	}
 }
 
-static void opj_t1_dec_clnpass(
+#define MACRO_t1_flags_internal(x,y,flags_stride) t1->flags[((x)*(flags_stride))+(y)]
+
+#define opj_t1_dec_clnpass_internal(consistency_check, t1, bpno, cblksty, w, h, flags_stride) \
```

Comment:
Hi, is it possible to use an inline function here so that debugging is made easier ? like what was done in ae1da37558796508abcaff73632fb2ab06decb9f

Timings will have to be checked after that of course.